Header header
int64 id
int64 timestamp
Hand[] hands
InteractionBox interaction_box
float64 current_frames_per_second
bool is_valid
